app/src/main/java/com/sdpteam/connectout/qr_code/QRcodeEventActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.sdpteam.connectout.qr_code;
+
+import androidx.activity.result.ActivityResult;
+import androidx.activity.result.ActivityResultCallback;
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.contract.ActivityResultContracts;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import com.sdpteam.connectout.R;
+
+public class QRcodeEventActivity extends AppCompatActivity {
```

Comment:
From what I understood this is an activity to create a QR code which is only used for the demo. Maybe you should add a TODO to remove it on next sprint when we will link the QR code to the event view